### PR TITLE
updated icarus page

### PR DESCRIPTION
**Pull request by newtklocok**

### CODE REVIEW

File: docs/Teams/ICARUS/ICARUS.jpgZone.Identifier
Comment (zeulewan) [inline docs/Teams/ICARUS/ICARUS.jpgZone.Identifier:1]: Hi Newt, please do not include these zone identifier files in your commit.

**Comment by zeulewan** [comment]
Hi Newt, I'm going to close this request because of the zone identifiers. My sincere apologies sir!

**Comment by zeulewan** [comment]
Also your commit messages should be like future tense like you are commanding something:
https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages

**Comment by newtklocok** [comment]
okey dokey sir i'll fix that up and send another one

**Pull request closed without merging**